app/assets/stylesheets/building_blocks.scss

```diff
@@ -0,0 +1,83 @@
+html,
+body {
+  background: #FAFAFA;
+}
+
+.card {
+  background: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.